client/my-sites/themes/themes-selection.jsx

```diff
@@ -247,6 +250,43 @@ class ThemesSelection extends Component {
 		return options;
 	};
 
+	getActionLabel = ( theme ) => {
+		return this.props.getActionLabel( theme );
+	};
+
+	getPremiumThemePrice = ( theme ) => {
+		return this.props.getPremiumThemePrice( theme );
+	};
+
+	getScreenshotUrl = ( theme, themeOptions ) => {
+		return this.props.getScreenshotUrl( theme, themeOptions );
+	};
+
+	isThemeActive = ( theme ) => {
+		return this.props.isThemeActive( theme );
+	};
+
+	isInstallingTheme = ( theme ) => {
+		return this.props.isInstallingTheme( theme );
+	};
```

Comment:
I'm missing something here.

Why are we wrapping these functions instead of passing the props through?

E.g. can't we do this?
```jsx
<ThemeBlock
  getActionLabel={ this.props.getActionLabel( theme ) }
</ThemeBlock>
```

Reply:
I had it like that but it triggers a re-render when the props from ThemeSelection are called.